react/features/video-layout/middleware.web.js

```diff
@@ -53,6 +54,8 @@ MiddlewareRegistry.register(store => next => action => {
         break;
     }
 
+    case PARTICIPANTS_PANE_CLOSE:
+    case PARTICIPANTS_PANE_OPEN:
```

Comment:
Curious as to why the chat pane is not here. Thoughts @muscat1 ?

Reply:
`VideoLayout.onResize` is called whenever `toggleChat` is dispatched and that retroactively calls `VideoLayout.resizeVideoArea`. I could look into moving that here as well, in order to keep it tidy, good idea!